test/components/disable-container.spec.js

```diff
@@ -0,0 +1,70 @@
+import DisableContainer from '../../src/components/disable-container.vue';
+import { mount } from '../util/lifecycle';
+
+const mountComponent = (options = {}) =>
+  mount(DisableContainer, {
+    props: { disabled: false, disabledMessage: '' },
+    slots: { default: '<button>Test Content</button>' },
+    ...options
+  });
+
+describe('DisableContainer', () => {
+  it('renders slot content', () => {
+    const component = mountComponent();
+    component.get('button').text().should.equal('Test Content');
+  });
+
+  describe('disabled prop', () => {
+    it('shows overlay when disabled is true', () => {
+      const component = mountComponent({ props: { disabled: true } });
+      component.find('.disable-container-overlay').should.exist;
```

Comment:
```suggestion
      component.find('.disable-container-overlay').exists().should.be.true;
```

I think in Vue Test Utils, `.find()` will always return an object. So its return value will always exist / it will always not be `null`, which means the `.should.exist` assertion will definitely pass. I don't think the assertion delegates to the `exists()` method.

The `.get()` on the next line will automatically assert that the element exists, so the assertion on this line could probably also just be omitted if you prefer.